components/mail/dovecot/dovecot.p5m

```diff
@@ -0,0 +1,1055 @@
+file path=usr/lib/dovecot/lib30_imap_zlib_plugin.a
+file path=usr/lib/dovecot/lib30_imap_zlib_plugin.so
+file path=usr/lib/dovecot/lib90_stats_plugin.a
+file path=usr/lib/dovecot/lib90_stats_plugin.so
+file path=usr/lib/dovecot/lib95_imap_stats_plugin.a
+file path=usr/lib/dovecot/lib95_imap_stats_plugin.so
+file path=usr/lib/dovecot/lib99_welcome_plugin.a
+file path=usr/lib/dovecot/lib99_welcome_plugin.so
+file path=usr/lib/dovecot/libdcrypt_openssl.so
+file path=usr/lib/dovecot/libdovecot-compression.a
+link path=usr/lib/dovecot/libdovecot-compression.so \
+    target=libdovecot-compression.so.0.0.0
+link path=usr/lib/dovecot/libdovecot-compression.so.0 \
+    target=libdovecot-compression.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-compression.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-dsync.a
+link path=usr/lib/dovecot/libdovecot-dsync.so target=libdovecot-dsync.so.0.0.0
+link path=usr/lib/dovecot/libdovecot-dsync.so.0 target=libdovecot-dsync.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-dsync.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-fts.a
+link path=usr/lib/dovecot/libdovecot-fts.so target=libdovecot-fts.so.0.0.0
+link path=usr/lib/dovecot/libdovecot-fts.so.0 target=libdovecot-fts.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-fts.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-lda.a
+link path=usr/lib/dovecot/libdovecot-lda.so target=libdovecot-lda.so.0.0.0
+link path=usr/lib/dovecot/libdovecot-lda.so.0 target=libdovecot-lda.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-lda.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-login.a
+link path=usr/lib/dovecot/libdovecot-login.so target=libdovecot-login.so.0.0.0
+link path=usr/lib/dovecot/libdovecot-login.so.0 target=libdovecot-login.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-login.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-sql.a
+link path=usr/lib/dovecot/libdovecot-sql.so target=libdovecot-sql.so.0.0.0
+link path=usr/lib/dovecot/libdovecot-sql.so.0 target=libdovecot-sql.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-sql.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-storage.a
+link path=usr/lib/dovecot/libdovecot-storage.so \
+    target=libdovecot-storage.so.0.0.0
+link path=usr/lib/dovecot/libdovecot-storage.so.0 \
+    target=libdovecot-storage.so.0.0.0
+file path=usr/lib/dovecot/libdovecot-storage.so.0.0.0
+file path=usr/lib/dovecot/libdovecot.a
+link path=usr/lib/dovecot/libdovecot.so target=libdovecot.so.0.0.0
+link path=usr/lib/dovecot/libdovecot.so.0 target=libdovecot.so.0.0.0
+file path=usr/lib/dovecot/libdovecot.so.0.0.0
+file path=usr/lib/dovecot/libfs_compress.a
+file path=usr/lib/dovecot/libfs_compress.so
+file path=usr/lib/dovecot/libssl_iostream_openssl.a
+file path=usr/lib/dovecot/libssl_iostream_openssl.so
+file path=usr/lib/dovecot/stats/libstats_auth.a
+file path=usr/lib/dovecot/stats/libstats_auth.so
+file path=usr/lib/dovecot/stats/libstats_mail.a
+file path=usr/lib/dovecot/stats/libstats_mail.so
+file path=usr/libexec/dovecot/aggregator
```

Comment:
I ended up using /usr/lib/dovecot/dovecot because the build system uses libexecdir/dovecot as the location for those bins.  The alternative is to patch the dovecot build system - and I'm not really a fan of carrying patches.